ereol_inspiration/ereol_inspiration.features.inc

```diff
@@ -13,6 +13,34 @@ function ereol_inspiration_ctools_plugin_api($module = NULL, $api = NULL) {
   }
 }
 
+/**
+ * Implements hook_image_default_styles().
+ */
+function ereol_inspiration_image_default_styles() {
+  $styles = array();
+
+  // Exported image style: app_feed_image.
```

Comment:
This image style is also found in https://github.com/eReolen/ereolengo/pull/7/files#diff-5546ba06abf6b3ce90a2fc5794b8ddfdR19

It should have been defined in the reol_app_feeds module as shared code and removed from the site modules.

Reply:
Fixed.